presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -169,37 +169,55 @@ export class PresentationRpcInterface extends RpcInterface {
 
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getContentDescriptor(_token: IModelRpcProps, _options: ContentDescriptorRpcRequestOptions): PresentationRpcResponse<DescriptorJSON | undefined> { return this.forward(arguments); }
+
   public async getContentSetSize(_token: IModelRpcProps, _options: ContentRpcRequestOptions): PresentationRpcResponse<number> { return this.forward(arguments); }
+
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getPagedContent(_token: IModelRpcProps, _options: Paged<ContentRpcRequestOptions>): PresentationRpcResponse<{ descriptor: DescriptorJSON, contentSet: PagedResponse<ItemJSON> } | undefined> { return this.forward(arguments); }
+
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getPagedContentSet(_token: IModelRpcProps, _options: Paged<ContentRpcRequestOptions>): PresentationRpcResponse<PagedResponse<ItemJSON>> { return this.forward(arguments); }
 
   /** @beta */
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getElementProperties(_token: IModelRpcProps, _options: SingleElementPropertiesRpcRequestOptions): PresentationRpcResponse<ElementProperties | undefined> { return this.forward(arguments); }
 
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getPagedDistinctValues(_token: IModelRpcProps, _options: DistinctValuesRpcRequestOptions): PresentationRpcResponse<PagedResponse<DisplayValueGroupJSON>> { return this.forward(arguments); }
 
   /** @beta */
   public async getContentInstanceKeys(_token: IModelRpcProps, _options: ContentInstanceKeysRpcRequestOptions): PresentationRpcResponse<{ total: number, items: KeySetJSON }> { return this.forward(arguments); }
 
   public async getDisplayLabelDefinition(_token: IModelRpcProps, _options: DisplayLabelRpcRequestOptions): PresentationRpcResponse<LabelDefinitionJSON> { return this.forward(arguments); }
+
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
   public async getPagedDisplayLabelDefinitions(_token: IModelRpcProps, _options: DisplayLabelsRpcRequestOptions): PresentationRpcResponse<PagedResponse<LabelDefinitionJSON>> { return this.forward(arguments); }
 
   public async getSelectionScopes(_token: IModelRpcProps, _options: SelectionScopeRpcRequestOptions): PresentationRpcResponse<SelectionScope[]> { return this.forward(arguments); }
+
   // TODO: need to enforce paging on this
   public async computeSelection(_token: IModelRpcProps, _options: SelectionScopeRpcRequestOptions, _ids: Id64String[], _scopeId: string): PresentationRpcResponse<KeySetJSON>;
   /** @alpha */
   public async computeSelection(_token: IModelRpcProps, _options: ComputeSelectionRpcRequestOptions): PresentationRpcResponse<KeySetJSON>;
+  @RpcOperation.setPolicy({ allowResponseCompression: true })
```

Comment:
Not so sure about this one - generally `KeySetJSON` sends IDs grouped by class names. The IDs are sent as a `CompressedId64Set`, so I believe making additional compression might not be worth it. Do you think otherwise?

Reply:
Depends on how many instance ids can potentially be returned by this operation. If it can easily get into thousands, then there's still a lot of room for compression because the data is still being represented in string form instead of binary.

Reply:
The frontend sends these `computeSelection` requests in batches of 5000 ids. Generally, when applying selection scopes (at least the ones that we currently have), we "walk up" rather than "down" - return a model, category or one of the ancestor elements, so we should always get 5000 or less instance keys as a result. Of course this could mean we have 5000 classes with one instance ID each, or we could have one class with 5000 compressed IDs. The sizes of these responses would be very different... 

On the other hand, I see what you mean by saying that even compressed IDs string would still compress well, since it's a string. Let's keep this as is.